### PR TITLE
#1: Extracted database cleaner from exasol-testcontainers

### LINKED ISSUE #1: Extract database cleaner from exasol-testcontainers
Extract database cleaner from https://github.com/exasol/exasol-testcontainers/

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #1

### CODE REVIEW

**Comment by jakobbraun** [comment]
CI fails because of the docker rate limit. Needs a restart on moday

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Exasol
```
Comment (morazow) [inline LICENSE]:
```suggestion
Copyright (c) 2021 Exasol
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+| [Maven Jacoco Plugin](https://www.eclemma.org/jacoco/trunk/doc/maven.html)          | Code coverage metering                                 | Eclipse Public License 2.0    |
+| [Maven Source Plugin](https://maven.apache.org/plugins/maven-source-plugin/)        | Creating a source code JAR                             | Apache License 2.0            |
+| [Maven Surefire Plugin](https://maven.apache.org/surefire/maven-surefire-plugin/)   | Unit testing                                           | Apache License 2.0            |
+| [Project Keeper Maven Plugin][project-keeper-maven-plugin]                          | Checking project structure                             | MIT License                   |
+| [Sonatype OSS Index Maven Plugin][sonatype-oss-index-maven-plugin]                  | Checking Dependencies Vulnerability                    | ASL2                          |
+| [Versions Maven Plugin][versions-maven-plugin]                                      | Checking if dependencies updates are available         | Apache License 2.0            |
+
+# @formatter:off
```
Comment (morazow) [inline README.md]: Remove these.

You can use html commenting `<!--  -->` in markdown if you need. But I think you can just remove them.

File: doc/changes/changelog.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Changes
+
+* [0.1.0](changes_0.1.0.md)
```
Comment (morazow) [inline doc/changes/changelog.md]:
```suggestion
* [1.0.0](changes_1.0.0.md)
```

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>database-cleaner</artifactId>
+    <version>1.0.0</version>
+    <name>database-cleaner</name>
+    <description>Tool that deletes all object in an Exasol database
```
Comment (morazow) [inline pom.xml]:
```suggestion
    <description>Tool that deletes all objects in an Exasol database
```

**Review by morazow — CHANGES_REQUESTED** [review]

**Comment by morazow** [comment]
So this is the part Sebastian was mentioning in the channel! Thanks, this looks great!

**Review by morazow — APPROVED** [review]

**Pull request merged**